workbench-core/accounts/src/services/hostingAccountService.ts

```diff
@@ -20,6 +21,10 @@ export default class HostingAccountService {
     return await new HostingAccountLifecycleService().getAccount(accountId, true);
   }
 
+  public async buildTemplateUrlsForAccount(externalId: string): Promise<TemplateResponse> {
+    return await new HostingAccountLifecycleService().buildTemplateUrlsForAccount(externalId);
```

Comment:
Should we be creating a new `HostingAccountLifecycleService` instance for each function?